easybuild/easyconfigs/p/python-igraph/python-igraph-0.10.3-foss-2022a.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = 'python-igraph'
+version = '0.10.3'
+
+homepage = 'https://igraph.org/python'
+description = """Python interface to the igraph high performance graph library, primarily aimed at complex network
+ research and analysis."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('igraph', '0.10.3'),
+    ('PyCairo', '1.21.0'),
+    ('Clang', '13.0.1'),
+    ('libxml2', '2.9.13'),
```

Comment:
When python-igraph is built using an external igraph library (instead of the vendored one), libxml2 is not a direct dependency. It is only an indirect dependency though igraph.